ios/engine/KMEI/KeymanEngineTests/QueryPackageVersionTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  QueryPackageVersionTests.swift
+//  KeymanEngineTests
+//
+//  Created by Joshua Horton on 6/25/20.
+//  Copyright © 2020 SIL International. All rights reserved.
+//
+
+import XCTest
+@testable import KeymanEngine
+
+class QueryPackageVersionTests: XCTestCase {
+  let PACKAGE_HOST_DOMAIN = "https://downloads.keyman.com/"
+  /**
+   * A rigorous test of our package-version query code that runs against the actual, live api.keyman.com endpoint.
```

Comment:
Why are you running against a live endpoint instead of a mock?

Reply:
I may well add a Mock, and I do plan to use it at times with one.  That stated, we do control this live endpoint; thought it might make a decent 'canary'.  We should be able to rely on it being up, especially since CI does require 'net access.

Also, this was the last bit of the PR I threw together at the end of the day; a mocked version would likely not have been completed until tomorrow.